tools/debian/rules

```diff
@@ -11,6 +11,9 @@ endif
 %:
 	dh $@ --with=systemd
 
+clean:
+	dh $@
+
```

Comment:
This is a very non-standard way to do this (to not say "hack"), and it's also potentially brittle. Either install the build dependencies, or skip the cleaning and build dependencies with `dpkg-buildpackage -S -us -uc -nc -d`. 

cockpit does not contain `debuild` any more, so I figure this is the invocation from cockpituous. Doing `-nc` (skip cleaning) there is perfectly safe as the directory just gets created from a clean unpack of the tarball, so there is nothing to clean.